components/automate-cli/pkg/verifyserver/services/softwareversionservice/softwareversionservice.go

```diff
@@ -183,20 +184,31 @@ func (sv *SoftwareVersionService) checkKernelVersion(kernelFilePath, osName stri
 	kernelVersion, err := getosutils.GetKernelVersion(kernelFilePath)
 	if err != nil {
 		sv.logger.Error("Enable to get OS Version as the file on the path does not exit: ", err)
-		return failureResponse(KERNAL_VERSION_CHECK, "Its not feasible to determine the Kernal version of the system", "Please run automate on the supported platforms."), err
+		return failureResponse(KERNEL_VERSION_CHECK, "Its not feasible to determine the Kernel version of the system", "Please run automate on the supported platforms."), err
 	}
-	sv.logger.Debug("Got the kernal version:", kernelVersion)
-	checkVersion, _ := strconv.ParseFloat(kernelVersion, 64)
+	sv.logger.Debug("Got the kernel version:", kernelVersion)
 	if strings.Contains(strings.ToLower(osName), strings.ToLower(AMAZON_LINUX)) {
-		if checkVersion == KERNAL_SUPPORTED_VERSION_AMAZON_LINUX {
-			return successResponse(KERNAL_VERSION_CHECK, "Linux kernal version is "+fmt.Sprintf("%.2f", checkVersion)), nil
+		checkVersion, _ := strconv.ParseFloat(kernelVersion, 64)
+		if checkVersion == KERNEL_SUPPORTED_VERSION_AMAZON_LINUX {
+			return successResponse(KERNEL_VERSION_CHECK, "Linux kernel version is "+fmt.Sprintf("%.2f", checkVersion)), nil
 		}
-		return failureResponse(KERNAL_VERSION_CHECK, "Linux kernel version is not equal to 5.10", "Use a linux version whose kernel version is equal to 5.10"), nil
+		return failureResponse(KERNEL_VERSION_CHECK, "Linux kernel version is "+kernelVersion+" which is not equal to 5.10", "Use a linux version whose kernel version is equal to 5.10"), nil
```

Comment:
```suggestion
		return failureResponse(KERNEL_VERSION_CHECK, "Linux kernel version is "+kernelVersion+" which is lesser that 5.10", "Use a linux version whose kernel version is equal or greater than to 5.10"), nil
```
@punitmundra  @jayvikramsharma1  Review if this would be appropriate messages

Reply:
updated like this >> `Linux kernel version is "+kernelVersion+" which is lower than 5.10", "Use a linux version whose kernel version is greater than or equals to 5.10`